x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -36,6 +37,11 @@ interface Interval {
   type: string;
 }
 
+export interface Actions {
+  interval: string;
```

Comment:
This parameter is called `throttle` in the API and it can accept only [certain values](https://www.elastic.co/guide/en/security/current/rules-api-create.html)

<img width="796" alt="Screenshot 2022-09-27 at 20 09 27" src="https://user-images.githubusercontent.com/7359339/192603764-c985760f-7ab3-403f-83f9-e030a5c84b79.png">

If you rebase against main, you can import a more strictly defined type for it from the package (thanks to @jpdjere it is now more strictly defined):

```ts
import type { Throttle } from '@kbn/securitysolution-io-ts-alerting-types';
```

We use it in `x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.ts`.